anime_downloader/util.py

```diff
@@ -46,15 +46,17 @@ def search(query):
         logging.error(click.style(str(e), fg='red'))
         sys.exit(1)
     click.echo(format_search_results(search_results))
-
+    if not search_results: 
+        print('No such Anime found')
```

Comment:
I didn't know how to show this as an error (still don't know how to use try and except exactly), so I just printed with an 'if' :P

Reply:
Use `logging.error`.

Reply:
Sorry but replacing print with logging.error later gives sys.exit(1) as invalid syntax. Was I supposed to use it inside and except clause? I just cant figure out how to use it. Which error do I use in
'''
 except //error: 
        logging.error('No such Anime found. Please ensure correct spelling.')
        sys.exit(1)
'''